packages/accounts-2fa/2fa-client.js

```diff
@@ -0,0 +1,78 @@
+import { Accounts } from "meteor/accounts-base";
+
+// Used in the various functions below to handle errors consistently
+const reportError = (error, callback) => {
+  if (callback) {
+    callback(error);
+  } else {
+    throw error;
+  }
+};
+
+/**
+ * @summary Verify if the user has 2FA enabled
+ * @locus Client
+ * @param {Object|String} selector Username, email or custom selector to identify the user.
+ * @param {Function} [callback] Optional callback. Called with a boolean on success that indicates whether the user has
+ *    or not 2FA enabled, or with a single `Error` argument on failure.
+ */
+Accounts.has2FAEnabled = (selector, callback) => {
+  Accounts.connection.call(
+    'has2FAEnabled',
+    selector,
+    callback,
+  );
+};
+
+/**
+ * @summary Generates a svg QR code and save secret on user
+ * @locus Client
+ * @param {String} appName Optional. It's the name of your app that will show up when the user scans the QR code.
+ * @param {Function} [callback] Optional callback.
+ *   Called with no arguments on success, or with a single `Error` argument
+ *   on failure.
+ */
+Accounts.generateSvgCodeAndSaveSecret = (appName, callback) => {
```

Comment:
`generateSvgCodeAndSaveSecret` this name is telling implementation details.

It doesn't matter that the image is an SVG. What is important here is that we generate a QR Code to enable 2FA.

I would call it: `generate2faActivationQrCode`

Reply:
done